doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+    "\n",
+    "The *test statistics* available in [Alibi Detect](https://github.com/SeldonIO/alibi-detect) can be broadly split into two categories; univariate and multivariate tests.\n",
+    "\n",
+    "* Univariate:\n",
+    "    * [Chi-Squared](methods/chisquaredrift.ipynb) (for categorical data)\n",
+    "    * [Kolmogorov-Smirnov](methods/ksdrift.ipynb)\n",
+    "\n",
+    "* Multivariate:\n",
+    "    * [Least-Squares Density Difference (LSDD)](methods/lsdddrift.ipynb)\n",
+    "    * [Maximum Mean Discrepancy (MMD)](methods/mmddrift.ipynb)\n",
+    "\n",
+    "When applied to multidimensional data with dimension $d$, the univariate tests are applied in a feature-wise manner. The obtained p-values for each feature are aggregated either via the [Bonferroni](https://mathworld.wolfram.com/BonferroniCorrection.html) or the [False Discovery Rate](http://www.math.tau.ac.il/~ybenja/MyPapers/benjamini_hochberg1995.pdf) (FDR) correction. The Bonferroni correction is more conservative and controls for the probability of at least one false positive. The FDR correction on the other hand allows for an expected fraction of false positives to occur. \n",
+    "\n",
+    "Since the univariate tests examine the feature-wise marginal distributions, they may miss drift in cases where the joint distribution over all $d$ features has changed, but the marginals have not. The multivariate tests, which use permutation testing to extract p-values, offer a more powerful alternative at the cost of greater complexity."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dimension reduction <a class=\"anchor\" id=\"dimension_reduction\"></a>\n",
+    "\n",
+    "Given an input dataset $\\mathbf{X}\\in \\mathbb{R}^{N\\times d}$, where $N$ is the number of observations and $d$ the number of dimensions, the aim is the reduce the data dimensionality from $d$ to $K$, where $K\\ll d$. The lower dimensional data $\\hat{\\mathbf{X}}\\in \\mathbb{R}^{N\\times K}$ is then monitored by the drift detector. The paper [Failing Loudly: An Empirical Study of Methods for Detecting Dataset Shift](https://arxiv.org/abs/1810.11953) presents a number of dimension reduction approaches, which can be broadly categorised under:\n",
+    "\n",
+    "1. Linear projections\n",
```

Comment:
Might be worth considering the following categories:
1. Linear projections
2. Non-linear projections, optionally learned (incl. both random and trained encoders)
3. Feature maps extracted from the model (incl. model output layer which would be the only thing to extract from a truly black box model which you can only query but don't know the internals of)